app/src/main/kotlin/goddamn/GodDamnService.kt

```diff
@@ -0,0 +1,29 @@
+package com.mbcdev.goddamn
+
+import android.media.MediaPlayer
+import android.os.Build
+import android.service.quicksettings.TileService
+import android.support.annotation.RequiresApi
+
+/**
+ * Just stop judging me. It's your fault Egan and Brenda.
+ *
+ * It wasn't my fault before, but some of it is now.
+ *
+ * Created by bcarroll on 13/11/2017.
+ */
+@RequiresApi(api = Build.VERSION_CODES.N)
+class GodDamnService : TileService() {
```

Comment:
Would like to test this :) 
there are some lifecycle methods in TileService that might help too

Reply:
Yeah it was mostly me being too lazy to plug out my dev phone and plug in my real phone. Will try it now.